### ISSUE TITLE
Pug / and space after end of string 

### ISSUE THREAD

**Issue by brightwiths** [emmetio/sublime-text-plugin#14]
In Emmet 2 I have that problem with Pug: / and space at the end of string. 
![Untitled](https://user-images.githubusercontent.com/54640696/71783928-890c0680-3013-11ea-8278-948495bfce9c.png)

**Comment by sergeche** [comment]
It’s added intentionally so you can start typing tag contents without adding space manually. And if you have plugin like EditorConfig, it will automatically trim extra whitespace when saving document.

But I personally don’t use Pug and don’t know if this behaviour is convenient

**Comment by brightwiths** [comment]
> It’s added intentionally so you can start typing tag contents without adding space manually. And if you have plugin like EditorConfig, it will automatically trim extra whitespace when saving document.
> 
> But I personally don’t use Pug and don’t know if this behaviour is convenient

Great thx! Editor config remove space at the end. And now need to remove **/** at the end of some strings. How it possible?

**Comment by sergeche** [comment]
You need [EditorConfig](https://packagecontrol.io/packages/EditorConfig) plugin. See [official we-site](https://editorconfig.org) with usage examples

**Comment by brightwiths** [comment]
> You need [EditorConfig](https://packagecontrol.io/packages/EditorConfig) plugin. See [official we-site](https://editorconfig.org) with usage examples

Hello. I wrote in previos post that I already used EditorConfig. It can remove spaces at end by:
`trim_trailing_whitespace = true`
But there is no way in this plugin to trim trailing slash at the end. I readed complete list of properties of EditorConfig, EditorConfig Specification and checked the examples of projects using EditorConfig files. 
Looking the way to remove / - trailing slash at the end of string.

**Comment by sergeche** [comment]
Trailing slash is used for self-closed elements like `<link />` or `<br />`, not sure if it should be removed

**Comment by brightwiths** [comment]
> Trailing slash is used for self-closed elements like `<link />` or `<br />`, not sure if it should be removed
Ok, but in previous version of Emmet 1, it was not. And look like no clean in pug code / at the end, when there are no <>
<!DOCTYPE html>
```
html(lang="en")
	head
		meta(charset="UTF-8")/
		meta(name="viewport", content="width=device-width, initial-scale=1.0")/
		title Document
	body
		img(src="", alt="")/
		a(href="#")
```

**Comment by sergeche** [comment]
Fixed in v0.1.8

**Comment by brightwiths** [comment]
> Fixed in v0.1.8

Brilliant, thx you! You make me happy with it)
I wish you be also happy and energetic in new year! =)
